### PR TITLE
Enable prow for Website repo

### PR DESCRIPTION
**Pull request by samodell**

### CODE REVIEW

**Comment by samodell** [comment]
Following the instructions [here](https://github.com/knative/test-infra/blob/master/ci/prow/prow_setup.md#setting-up-prow-for-a-new-repo).

After this merges, can a test-infra owner run `make update-config`?

Thanks!

cc @RichieEscarez

**Comment by samodell** [comment]
And one question -- the last step in the instructions says, "Set tide as a required status check for the master branch." How is that done?

**Comment by adrcunha** [comment]
> And one question -- the last step in the instructions says, "Set tide as a required status check for the
> master branch." How is that done?

The repo admin needs to go to the settings and enable the checks (there's a section in the settings for that). The jobs will only appear there once at least one PR triggers them.

File: ci/prow/config_knative.yaml [hunk 1]
```diff
@@ -95,6 +95,11 @@ presubmits:
     - integration-tests: true
     - go-coverage: true
       go-coverage-threshold: 50
+      
+  knative/website:
```
Comment (adrcunha) [inline ci/prow/config_knative.yaml]: Please run `make config` and add `config.yaml` to this PR.

**Comment by adrcunha** [comment]
> Following the instructions here

/hold

@samodell I just realized that presubmits don't apply to the website repo. You probably just want to enable automatic merges (respecting OWNERS) using Prow, right?

**Comment by samodell** [comment]
@adrcunha yes, automatic merges, and I think the CLA check as well.

**Comment by samodell** [comment]
Ah, so I probably should be editing the periodics section below, and not the presubmits? I wasn't sure.

**Comment by adrcunha** [comment]
I'll take care of this, don't worry.

**Pull request closed without merging**